index.js

```diff
@@ -0,0 +1,120 @@
+const myLibrary = [];
+
+const formContainer = document.getElementById('form_container');
+const bookShelf = document.getElementById('book_shelf');
+
+let bookID = myLibrary.length - 1;
+function Book(title, pages) {
+  this.id = bookID;
+  bookID += 1;
+
+  this.title = title;
+  this.pages = pages;
+  this.read = false;
+}
+
+function deleteBook(bookContainer, book) {
+  bookContainer.parentNode.removeChild(bookContainer);
+  myLibrary.splice(book.id, 1);
+}
+
+function readBook(readButton, book) {
+  if (book.read === false) {
+    readButton.style.backgroundColor = 'green';
+  }
+  if (book.read === true) {
+    readButton.style.backgroundColor = 'red';
+  }
+  book.read = !book.read;
+}
+
+function addBook(book) {
+  const bookContainer = document.createElement('div');
+  bookContainer.setAttribute('class', 'bookContainer');
+  bookContainer.setAttribute('id', '');
+  bookShelf.appendChild(bookContainer);
+
```

Comment:
- Separate JS logic from DOM manipulation

Reply:
We do not understand this example. Please elaborate on another one, since all of the lines you reference contain DOM element manipulations.